### ISSUE TITLE
Error printing event in event handler

### ISSUE THREAD

**Issue by ryanjoneil** [scipopt/PySCIPOpt#766]
**Describe the bug**
I get an error when I attempt to print or otherwise inspect the `event` argument of the `Eventhdlr.eventexec` method.

**To Reproduce**

Here is a simple example:
```python
from pyscipopt import quicksum as sum
import pyscipopt as scip
import random

class handler(scip.Eventhdlr):
    def eventexec(self, event):
        print(event)

    def eventinit(self):
        self.model.catchEvent(scip.SCIP_EVENTTYPE.LPEVENT, self)

    def eventexit(self):
        self.model.dropEvent(scip.SCIP_EVENTTYPE.LPEVENT, self)

if __name__ == '__main__':
    m = scip.Model()
    m.includeEventhdlr(handler(), 'handler', 'decription')

    x = [m.addVar(obj=i) for i in range(1000)]
    for _ in range(1000):
        left = sum(random.sample(x, random.randint(5, 15)))
        right = random.randint(1, 100)
        m.addCons(left >= right)

    m.optimize()
```

When I run it I get this:

```txt
feasible solution found by trivial heuristic after 0.0 seconds, objective value 4.995000e+10
presolving:
(round 1, fast)       1 del vars, 7 del conss, 0 add conss, 999 chg bounds, 0 chg sides, 0 chg coeffs, 0 upgd conss, 0 impls, 0 clqs
(round 2, fast)       1 del vars, 7 del conss, 0 add conss, 1998 chg bounds, 0 chg sides, 0 chg coeffs, 0 upgd conss, 0 impls, 0 clqs
   (0.0s) running MILP presolver
   (0.0s) MILP presolver found nothing
   (0.0s) symmetry computation started: requiring (bin +, int +, cont +), (fixed: bin -, int -, cont -)
   (0.0s) no symmetry present (symcode time: 0.00)
presolving (3 rounds: 3 fast, 1 medium, 1 exhaustive):
 1 deleted vars, 7 deleted constraints, 0 added constraints, 1998 tightened bounds, 0 added holes, 0 changed sides, 0 changed coefficients
 0 implications, 0 cliques
presolved problem has 999 variables (0 bin, 0 int, 0 impl, 999 cont) and 993 constraints
    993 constraints of type <linear>
Presolving Time: 0.01

 time | node  | left  |LP iter|LP it/n|mem/heur|mdpt |vars |cons |rows |cuts |sepa|confs|strbr|  dualbound   | primalbound  |  gap   | compl. 
t 0.0s|     1 |     0 |     0 |     - | trivial|   0 | 999 | 993 |   0 |   0 |  0 |   0 |   0 | 0.000000e+00 | 4.512101e+07 |    Inf | unknown
Traceback (most recent call last):
  File "/home/ryan/src/scratch/bar.py", line 7, in eventexec
    print(event)
TypeError: __str__ returned non-string (type int)
Exception ignored in: 'pyscipopt.scip.PyEventExec'
Traceback (most recent call last):
  File "/home/ryan/src/scratch/bar.py", line 7, in eventexec
    print(event)
TypeError: __str__ returned non-string (type int)
[event.c:313] ERROR: Error <0> in function call
[event.c:2122] ERROR: Error <0> in function call
[event.c:1616] ERROR: Error <0> in function call
[solve.c:1523] ERROR: Error <0> in function call
[solve.c:2979] ERROR: Error <0> in function call
[solve.c:3898] ERROR: Error <0> in function call
[solve.c:4198] ERROR: Error <0> in function call
[solve.c:4994] ERROR: Error <0> in function call
[scip_solve.c:2789] ERROR: Error <0> in function call
Traceback (most recent call last):
  File "/home/ryan/src/scratch/bar.py", line 25, in <module>
    m.optimize()
  File "src/pyscipopt/scip.pxi", line 3355, in pyscipopt.scip.Model.optimize
  File "src/pyscipopt/scip.pxi", line 256, in pyscipopt.scip.PY_SCIP_CALL
Exception: SCIP: unspecified error!
```

I get similar errors if I call other functions like `type(event)` or `str(event)`.

**Expected behavior**
I was hoping to get more information about the `event`.

**System**
 - OS: Fedora 39 on amd64
 - Version: PySCIPOpt 4.4.0
 - SCIP version: 8.1.0
 - How did you install `pyscipopt`? `pip`

**Comment by Joao-Dionisio** [comment]
Hello, @ryanjoneil! I can't look into it a lot right now, but I didn't want to leave you hanging. The events are just unsigned ints that SCIP then decides what to do with them.

```print(event.getType())```  gives you the number associated with the event, but this wouldn't give you much information. Probably we should make it possible to get the name of the event, yes. Right now I don't know how to do it, so I'd need some time to learn.

I was able to use ```print(type(event))```, it prints ```<class 'pyscipopt.scip.Event'>```.

**Comment by ryanjoneil** [comment]
> Hello, @ryanjoneil! I can't look into it a lot right now, but I didn't want to leave you hanging. The events are just unsigned ints that SCIP then decides what to do with them.
> 
> `print(event.getType())` gives you the number associated with the event, but this wouldn't give you much information. Probably we should make it possible to get the name of the event, yes. Right now I don't know how to do it, so I'd need some time to learn.
> 
> I was able to use `print(type(event))`, it prints `<class 'pyscipopt.scip.Event'>`.

That does work - I don't know how to update the Cython code and open a PR, but this does what I want in case that helps:

```python
from pyscipopt import quicksum as sum
import pyscipopt as scip
import random

EVENTS = {}
for name in dir(scip.SCIP_EVENTTYPE):
    attr = getattr(scip.SCIP_EVENTTYPE, name)
    if isinstance(attr, int):
        EVENTS[attr] = name

class handler(scip.Eventhdlr):
    def eventexec(self, event):
        print(EVENTS[event.getType()])

    def eventinit(self):
        self.model.catchEvent(scip.SCIP_EVENTTYPE.LPEVENT, self)

    def eventexit(self):
        self.model.dropEvent(scip.SCIP_EVENTTYPE.LPEVENT, self)

if __name__ == '__main__':
    m = scip.Model()
    m.includeEventhdlr(handler(), 'handler', 'decription')

    x = [m.addVar(obj=i) for i in range(1000)]
    for _ in range(1000):
        left = sum(random.sample(x, random.randint(5, 15)))
        right = random.randint(1, 100)
        m.addCons(left >= right)

    m.optimize()
```

**Comment by Joao-Dionisio** [comment]
Mostly fixed in #769.
